src/Path/Include.hs

```diff
@@ -377,6 +381,31 @@ parent (Path fp) =
     $ FilePath.takeDirectory
     $ FilePath.dropTrailingPathSeparator fp
 
+-- | Split an absolute path into a drive and, perhaps, a path. On POSIX, @/@ is
+-- a drive.
```

Comment:
>  On POSIX, @/@ is a drive.

Is it a good idea to have these functions defined for non-windows?
We could put it in the windows-specific code I guess, but I don't have an opinion yet about whether that's a good idea.

Reply:
Non-well typed equivalent functions exist for both POSIX and WINDOWS at `filepath` package: [`System.FilePath`](https://hackage.haskell.org/package/filepath-1.4.100.4/docs/System-FilePath.html#g:5). I think it is sensible to prove well-typed equivalents. I followed that package's Haddock documentation for this package.